packages/ui/src/components/molecules/Table/TableBox.tsx

```diff
@@ -0,0 +1,191 @@
+  HeightProps,
+  spacing,
+  SpacingProps,
+  width,
+  WidthProps,
+} from '../../utils';
+
+export interface TableBoxUtilityProps
+  extends WidthProps,
+    HeightProps,
+    FlexProps,
+    DisplayProps,
+    SpacingProps {}
+
+interface TableHeaderDataProps extends TableBoxUtilityProps {
+  data: string;
+  $ascending: boolean;
+  onClick: (name: string) => void;
+  selected?: boolean;
+  $noFlex?: boolean;
+}
+
+interface TableBoxDataRowProps extends TableBoxUtilityProps {
+  $last?: boolean;
+  $noFlex?: boolean;
+}
+
+const TableBoxStyle = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  border-width: 1px;
+  border-style: solid;
+  border-radius: 24px;
+  background-image: ${({ theme }) => theme.palette.background.primary};
+  box-shadow: ${({ theme }) => theme.shadow.popup};
+  border-color: ${({ theme }) => theme.palette.border.popup};
+  text-align: center;
+  ${flex}
+  ${width}
+  ${height}
+  ${spacing}
+`;
+
+const TableBoxTitleStyle = styled.div`
+  border-bottom: 1px;
+  border-top: 0;
+  border-left: 0;
+  border-right: 0;
+  border-style: solid;
+  padding: 16px 40px;
+  border-color: ${({ theme }) => theme.palette.border.popup};
+  color: ${({ theme }) => theme.palette.text.mutedText};
+  ${flex}
+  ${width}
+  ${height}
+  ${spacing}
+`;
+
+const TableHeaderStyle = styled.div`
+  display: flex;
+  align-items: flex-start;
+  align-self: flex-start;
+  border-bottom: 1px solid ${({ theme }) => theme.palette.border.table};
+  justify-content: space-between;
+  background-color: ${({ theme }) => theme.palette.background.input};
+  ${flex}
+  ${width}
+  ${height}
+  ${spacing}
+`;
+
+const TableHeaderDataStyle = styled.div<TableHeaderDataProps>`
+  display: flex;
+  flex: 1;
+  align-items: center;
+  justify-content: space-between;
+  flex: ${({ $noFlex }) => ($noFlex ? 'unset' : '1')};
+  padding: 16px 20px 16px 40px;
+  ${flex}
+  ${width}
+  ${height}
+  ${spacing}
+`;
+
+const TableBodyStyle = styled.div`
+  display: flex;
+  align-items: flex-start;
+  align-self: flex-start;
+  flex-direction: column;
+  border-bottom: 1px solid ${({ theme }) => theme.palette.border.table};
+  justify-content: space-between;
+  background-color: ${({ theme }) => theme.palette.background.input};
+  border-bottom-right-radius: 24px;
+  border-bottom-left-radius: 24px;
+  ${flex}
+  ${width}
+  ${height}
+  ${spacing}
+`;
+
+const TableDataRowStyle = styled.div<TableBoxDataRowProps>`
+  display: flex;
+  flex: 1;
+  align-items: center;
+  justify-content: space-between;
+  flex: ${({ $noFlex }) => ($noFlex ? 'unset' : '1')};
+  border-bottom-right-radius: ${({ $last }) => ($last ? '24px' : '0')};
+  border-bottom-left-radius: ${({ $last }) => ($last ? '24px' : '0')};
+  background: ${({ theme }) => theme.palette.background.content};
+
+  ${flex}
+  ${width}
+  ${height}
+  ${spacing}
+  &:hover {
+    background: ${({ theme }) => theme.palette.background.sideBar};
+  }
```

Comment:
Utilities should be at the end
```suggestion
  &:hover {
    background: ${({ theme }) => theme.palette.background.sideBar};
  }
  ${flex}
  ${width}
  ${height}
  ${spacing}
```